specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/CreateBillingProfile.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "billingAccountName": "{billingAccountName}",
+    "parameters": {
+      "displayName": "Finance",
+      "address": {
+        "firstName": "Test",
+        "lastName": "User",
+        "addressLine1": "1 Microsoft Way",
+        "city": "Redmond",
+        "postalCode": "98052",
+        "region": "WA",
+        "country": "US"
+      },
+      "enableAzureSkuIds": [
+        "0001",
+        "0002"
+      ],
+      "invoiceEmailOptIn": true,
+      "poNumber": "ABC12345",
+      "createAzureSubscriptions": true,
+      "paymentInstrumentId": "21dd9edc-af71-4d62-80ce-37151d475326"
```

Comment:
This should be an ARM id, not an ARM name.

Reply:
paymentinstrumentId is no longer required as we can infer that from billingaccount since there will always be only single checkwire paymentmethod shared by all billing profiles.